File: app/src/main/java/io/github/howiezuo/unsplash/model/Photo.java
package io.github.howiezuo.unsplash.model;

import com.google.gson.annotations.SerializedName;

public class Photo {

    String id;

    int width;

    int height;

    String color;

    int likes;

    @SerializedName("liked_by_user")
    boolean likedByUser;

    Urls urls;

    public String getId() {
        return id;
    }

    public Urls getUrls() {
        return urls;
    }
}

File: app/src/main/java/io/github/howiezuo/unsplash/model/Photo.kt
package io.github.howiezuo.unsplash.model

import com.google.gson.annotations.SerializedName


data class Photo(
        val id: String,
        val width: Int,
        val height: Int,
        val color: String,
        val likes: Int,
        @SerializedName("liked_by_user") val likedByUser: Boolean,
        val urls: Urls
)
